tests/block/k8s/integration/k8sBlockStorageIntegration_test.go

```diff
@@ -57,7 +61,13 @@ func (s *K8sBlockEnd2EndIntegrationSuite) SetupSuite() {
 
 	var err error
 
-	s.testClient, err = clients.CreateTestClient(tests.Platform)
+	rook, err := installer.NewK8sRookhelper()
```

Comment:
it looks like the global `rook` var will never be set. You're re-defining it here with this assignment. Does `TearDownSuite()` not give a null ref? I would suggest a more specific variable name than `rook` as well. What about `kubeContext`?